release-notes/v1_6.md

```diff
@@ -186,6 +203,7 @@ We will work on bringing these features back during the next iteration.
 
 However, we are not done yet.. Now that we have extracted languages into extensions, the next challenge is to support the nesting of these languages, that is, embed JavaScript or CSS inside HTML. We will be working on this in October.
 
+
 ## Extensions
 
 ### Workspace Recommended Extensions
```

Comment:
Really should add in the why here - we added this so that as athe maintainer of a project you can suggest the best extensions to use with it....